Packs/ServiceNow/Playbooks/playbook-ServiceNow_-_Ticket_Management.yml

```diff
@@ -0,0 +1,843 @@
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+                    ignorecase: true
+                accessor: brand
+            iscontext: true
+          right:
+            value:
+              simple: ServiceNow v2
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -150,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: aa176873-8f95-45b9-857d-6db87bdcef37
+    type: regular
+    task:
+      id: aa176873-8f95-45b9-857d-6db87bdcef37
+      version: -1
+      name: Create a ticket for the incident
+      description: Creates new ServiceNow ticket.
+      script: '|||servicenow-create-ticket'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      assignment_group:
+        complex:
+          root: inputs.serviceNowAssignmentGroup
+      category:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "1"
+      comments:
+        complex:
+          root: inputs.CommentToAdd
+      description:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "0"
+      impact:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "2"
+      severity:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "4"
+      short_description:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "6"
+      ticket_type:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "5"
+      urgency:
+        complex:
+          root: Please provide the following details to open a ticket.Answers
+          accessor: "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 1315
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: c45fcc66-2992-470f-8810-2ac6f6fa4640
+    type: condition
+    task:
+      id: c45fcc66-2992-470f-8810-2ac6f6fa4640
+      version: -1
+      name: whether to open a ticket for this incident?
+      description: Checks whether to open a new ticket or add a comment to the task.
```

Comment:
"..or add a comment to the task" ?

Reply:
Done. changed to
Checks whether to open a new ticket or add a comment to an existing ticket.